### PR TITLE
Tweaks to the git2r section

### PR DESCRIPTION
**Pull request by jennybc**
I tried to make separate commits because sometimes it's hard to see what changed in prose. And you can't tell how well it will be displayed until you make the PR.

### CODE REVIEW

File: README.md
Comment (jennybc) [inline README.md]: In theory, git2r can discover credentials. That probably only applies to ssh and, even then, as we know, often did not work. But I still think "may" is the right word here.

File: README.md
Comment (jennybc) [inline README.md:89]: In the next section about Simplicity, I sort of wanted to add more specific wording about sticking with implicit/explicit file paths, remote and branch names, etc. as opposed to using S3 or S4 objects. If you like that idea, I leave it to you to incorporate.

File: README.md
Comment (jennybc) [inline README.md:114]: Possible rewording or addition for clarity: Therefore, once you have credentials working in your command line Git life, you should also enjoy success when using gert.

File: README.md
Comment (jennybc) [inline README.md]: This is just about using the same term everywhere.

**Comment by jennybc** [comment]
One meta point that doesn't seem to be here yet:

When things go wrong with git2r (at install time re: ssh or at run time re: ssh or credential-finding), it is almost impossible to debug. There's very little visibility into how things are (not) working.

gert is very assertive about describing what it's doing and how that's going. It will hopefully "just work" more often and, even when it does not, it will be easier to troubleshoot.

**Comment by jennybc** [comment]
Nitpick: the capitalization of "gert" is not consistent.

**Review by jeroen — APPROVED** [review]

**Comment by jennybc** [comment]
I made a few more changes @jeroen before I merged, so give it a read. But I think they are all things you expected / implicitly approved.

**Comment by jeroen** [comment]
Thanks!

**Pull request merged**
